.travis.yml

```diff
@@ -44,21 +41,16 @@ matrix:
 before_install:
   - phpenv config-rm xdebug.ini
   - cd ../..
-  - composer selfupdate
-  - composer create-project -n --no-dev moodlerooms/moodle-plugin-ci ci ^1
+  - composer create-project -n --no-dev --prefer-dist moodlehq/moodle-plugin-ci ci ^3;
   - export PATH="$(cd ci/bin; pwd):$(cd ci/vendor/bin; pwd):$PATH"
 
 install:
-  - moodle-plugin-ci install
+  - moodle-plugin-ci install -vvv
 
 script:
   - moodle-plugin-ci phplint
   - moodle-plugin-ci phpcpd
   # - moodle-plugin-ci phpmd
   - moodle-plugin-ci codechecker
-  - moodle-plugin-ci csslint    # No CSS
```

Comment:
can we turn these back on now instead of removing them completely?

Reply:
These are dead in the new version, they get replaced by grunt, but that is broken due to es5/es6, and would fail on slick. I will see if there is another option